tox.ini

```diff
@@ -30,30 +29,20 @@ commands =
 deps =
 # for testing the typing module
     typing
-# numpydoc for typing scipy stack
-    numpydoc
-    {[testenv]deps}
-[testenv:py33]
-deps =
-    typing
     {[testenv]deps}
 [testenv:py34]
 deps =
     typing
-    numpydoc
     {[testenv]deps}
 [testenv:py35]
 deps =
-    numpydoc
     {[testenv]deps}
 [testenv:py36]
 deps =
-    numpydoc
     {[testenv]deps}
 [testenv:cov]
 deps =
     coverage
-    numpydoc
```

Comment:
Why is numpydoc removed here (and above)?

Reply:
Oops, that'll be a mistake from rebasing. I'll revert it!

Reply:
Thanks for finding this one!